### PR TITLE
Add Left 4 Dead and Left 4 Dead 2

### PR DESCRIPTION
**Pull request by gmazoyer**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [x] Have you added the server to the main README.md?
3. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

File: game_eggs/steamcmd_servers/left4dead/egg-left4dead.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-10-13T19:07:48+02:00",
+    "name": "Left 4 Dead",
+    "author": "pterodactyl@mazoyer.eu",
+    "description": "An outbreak of a highly contagious pathogen nicknamed the \"Green Flu\" begins in Pennsylvania. Two weeks after the first infection, four immune survivors, Bill, Zoey, Louis, and Francis make their way out of the city of Fairfield, only to discover that the infection is creating dangerous mutations in some of its hosts.",
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:source"
```
Comment (gOOvER) [inline game_eggs/steamcmd_servers/left4dead/egg-left4dead.json:13]: pls use new yolks image:
ghcr.io/parkervcp/games:source

File: game_eggs/steamcmd_servers/left4dead/egg-left4dead.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        "startup": "{\r\n    \"done\": \"Connection to Steam servers successful.\",\r\n    \"userInteraction\": []\r\n}",
+        "logs": "{\r\n    \"custom\": true,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "quit"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# SRCDS Base Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt -y update\r\napt -y --no-install-recommends install curl lib32gcc-s1 ca-certificates\r\n\r\ncd \/tmp\r\ncurl -sSL -o steamcmd.tar.gz http:\/\/media.steampowered.com\/installer\/steamcmd_linux.tar.gz\r\n\r\nmkdir -p \/mnt\/server\/steamcmd\r\ntar -xzvf steamcmd.tar.gz -C \/mnt\/server\/steamcmd\r\ncd \/mnt\/server\/steamcmd\r\n\r\n# SteamCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\n\r\nexport HOME=\/mnt\/server\r\n.\/steamcmd.sh +login anonymous +force_install_dir \/mnt\/server +app_update ${SRCDS_APPID} +quit\r\n\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so",
+            "container": "debian:bullseye",
```
Comment (gOOvER) [inline game_eggs/steamcmd_servers/left4dead/egg-left4dead.json:26]: pls use the yolks installer

ghcr.io/pterodactyl/installers:debian

File: game_eggs/steamcmd_servers/left4dead_2/egg-left4dead_2.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-10-13T23:28:59+02:00",
+    "name": "Left 4 Dead 2",
+    "author": "pterodactyl@mazoyer.eu",
+    "description": "Left 4 Dead 2 is set in the aftermath of a worldwide pandemic of a disease nicknamed the \"Green Flu\", which rapidly transforms humans into zombie-like creatures and mutated forms that demonstrate extreme aggression towards non-infected beings. A few humans are immune to the disease, while some of those who are infected have no symptoms. The Civil Emergency and Defense Agency (CEDA) and the U.S. military create safe zones to attempt to evacuate as many survivors as possible.",
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:source"
```
Comment (gOOvER) [inline game_eggs/steamcmd_servers/left4dead_2/egg-left4dead_2.json:13]: same her: 

ghcr.io/parkervcp/games:source

File: game_eggs/steamcmd_servers/left4dead_2/egg-left4dead_2.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        "startup": "{\r\n    \"done\": \"Connection to Steam servers successful.\",\r\n    \"userInteraction\": []\r\n}",
+        "logs": "{\r\n    \"custom\": true,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "quit"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# SRCDS Base Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt -y update\r\napt -y --no-install-recommends install curl lib32gcc-s1 ca-certificates\r\n\r\ncd \/tmp\r\ncurl -sSL -o steamcmd.tar.gz http:\/\/media.steampowered.com\/installer\/steamcmd_linux.tar.gz\r\n\r\nmkdir -p \/mnt\/server\/steamcmd\r\ntar -xzvf steamcmd.tar.gz -C \/mnt\/server\/steamcmd\r\ncd \/mnt\/server\/steamcmd\r\n\r\n# SteamCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\n\r\nexport HOME=\/mnt\/server\r\n.\/steamcmd.sh +login anonymous +force_install_dir \/mnt\/server +app_update ${SRCDS_APPID} +quit\r\n\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so",
+            "container": "debian:bullseye",
```
Comment (gOOvER) [inline game_eggs/steamcmd_servers/left4dead_2/egg-left4dead_2.json:26]: also use the yolks installer:
ghcr.io/pterodactyl/installers:debian

**Review by gOOvER — CHANGES_REQUESTED** [review]

**Review by gOOvER — CHANGES_REQUESTED** [review]
Please update to new Yolks as mentoined in the comments

**Comment by gmazoyer** [comment]
Updated with the mentioned images. Re-tested the servers and they still work as expected.

**Comment by parkervcp** [comment]
the steamcmd generic egg probably could have done this. I am not against it. I am just calling it out.

**Comment by gmazoyer** [comment]
Indeed you are right. It's simple enough to setup with the generic egg. Feel free to close this MR.

**Review by parkervcp — APPROVED** [review]

**Pull request merged**